CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Changelog
 
+## [2.3.2](https://launchpad.net/mixxx/+milestone/2.3.3) (unreleased)
+
+* Pioneer-DDJ-SB3: fix shift button release [#4659](https://github.com/mixxxdj/mixxx/pull/4659)
+* Traktor S3: Push two deck switches to explicitly clone decks [#4665](https://github.com/mixxxdj/mixxx/pull/4665) [#4671](https://github.com/mixxxdj/mixxx/pull/4671) [lp:1960680](https://bugs.launchpad.net/mixxx/+bug/1960680)
+* Behringer DDM4000: Update controller mapping [#4318](https://github.com/mixxxdj/mixxx/pull/4318)
+* MC7000: Fix 'inverted shift' bug in the controller mapping [#4755](https://github.com/mixxxdj/mixxx/pull/4755)
+* Fix spinback and break effect in the controller engine [#4708](https://github.com/mixxxdj/mixxx/pull/4708)
+* Fix scratch on first wheel touch [#4761](https://github.com/mixxxdj/mixxx/pull/4761) [lp:1800343](https://bugs.launchpad.net/mixxx/+bug/1800343)
+* Pref controller: prevent false-positive 'dirty' flag [#4721](https://github.com/mixxxdj/mixxx/pull/4721) [lp:1920844](https://bugs.launchpad.net/mixxx/+bug/1920844)
+* Fix rare SEGFAULT when closing the progress dialog [#4695](https://github.com/mixxxdj/mixxx/pull/4695)
```

Comment:
```suggestion
* Fix rare crash when closing the progress dialog [#4695](https://github.com/mixxxdj/mixxx/pull/4695)
```

No regular user knows what a segfault is, at least not on windows or macOS.

Reply:
"Stability improvements and fixes." Just copy it from any other changelog. :rofl: